salt/minion.py

```diff
@@ -1415,75 +1233,6 @@ def _refresh_grains_watcher(self, refresh_interval_in_minutes):
                     }
             })
 
-    def _set_tcp_keepalive(self):
-        if hasattr(zmq, 'TCP_KEEPALIVE'):
-            self.socket.setsockopt(
-                zmq.TCP_KEEPALIVE, self.opts['tcp_keepalive']
-            )
-            self.socket.setsockopt(
-                zmq.TCP_KEEPALIVE_IDLE, self.opts['tcp_keepalive_idle']
-            )
-            self.socket.setsockopt(
-                zmq.TCP_KEEPALIVE_CNT, self.opts['tcp_keepalive_cnt']
-            )
-            self.socket.setsockopt(
-                zmq.TCP_KEEPALIVE_INTVL, self.opts['tcp_keepalive_intvl']
-            )
-
-    def _set_monitor_socket(self):
```

Comment:
Did this functionality get re-implemented? I'm still finding my way around these changes but I don't see it.